examples/lustre/Dockerfile

```diff
@@ -0,0 +1,21 @@
+# ch-test-scope: full
+# ch-test-arch-exclude: aarch64
+# No lustre RPMS for aarch64
+FROM centos7
+
+# install development tools
+RUN yum install -y epel-release
+
+# more dependencies
+RUN yum -y install libyaml-devel \
+                   wget \
+                   gcc
+
+ENV LUSTRE_URL=https://downloads.whamcloud.com/public/lustre/latest-release/el7/client/RPMS/x86_64
+
+RUN wget ${LUSTRE_URL}/lustre-client-2.12.3-1.el7.x86_64.rpm && \
+    wget ${LUSTRE_URL}/kmod-lustre-client-2.12.3-1.el7.x86_64.rpm
```

Comment:
Format as:

```

RUN    wget ${LUSTRE_URL}/lustre-client-2.12.3-1.el7.x86_64.rpm \
    && wget ${LUSTRE_URL}/kmod-lustre-client-2.12.3-1.el7.x86_64.rpm
```

(This is so the `&&` doesn't get lost at the end of the line.)